src/components/Products.js

```diff
@@ -0,0 +1,34 @@
+// PRODUCTS COMPONENT //
+
+import { Swiper, SwiperSlide } from 'swiper/react';
+import 'swiper/css';
+import 'swiper/css/navigation';
+import { Navigation } from 'swiper/modules';
+
+export default function Products({itemsForDisplay}) {
+  
+  const cards = itemsForDisplay.map(item => {
+    return (
+        <SwiperSlide key={item.name}>
```

Comment:
I do know `item.name` and `item.id` resolve to the same thing but for readability, could we assign `item.id` to key? `id` can be assumed to be unique. (i know this is almost futile but easier to read).

Reply:
I agree with this; I think it's best practice